api/src/pcapi/utils/module_loading.py

```diff
@@ -3,11 +3,13 @@
 
 
 # Taken from django.utils.module_loading
-def import_string(dotted_path: str) -> Type:
+def import_string(dotted_path: str | None) -> Type:
     """
     Import a dotted module path and return the attribute/class designated by the
     last name in the path. Raise ImportError if the import failed.
     """
+    if dotted_path is None:
+        raise ImportError("Import path cannot be None")
```

Comment:
(détail) None ou chaîne vide, ça ne marchera dans aucun cas : 

```suggestion
    if not dotted_path:
        raise ImportError("Import path cannot be empty")
```